examples/custom-environment/README.md

```diff
@@ -0,0 +1,19 @@
+# `call-runtime` example
```

Comment:
I believe the description was accidentally copied from the previous PR :)

Reply:
:see_no_evil: changed